src/services/StoryServices.js

```diff
@@ -0,0 +1,25 @@
+import apiClient from "./services";
+
+export default {
+  getStories() {
+    return apiClient.get("stories");
+  },
+  getStoriesByUserId(userId) {
+    return apiClient.get(`stories/${userId}`);
+  },
+  getStory(id) {
+    return apiClient.get(`stories/${id}`);
+  },
+  addStory(userId) {
+    return apiClient.post(`stories/${userId}`);
```

Comment:
where does the body of the POST come from?

Reply:
This is mostly just a stub and it's not used anywhere. I believe Maurice will have an implementation incoming with a body.